saleor/product/utils/product_variants.py

```diff
@@ -0,0 +1,26 @@
+from django.core.exceptions import ValidationError
+
+from ..error_codes import ProductErrorCode
+
+
+def product_variant_exist(product):
```

Comment:
Also, the name could be more descriptive, something like `check_if_variant_exists`? Not perfect since the function not only checks that but also raises an error. Maybe this function should just do the check and return a bool, while error should be raised in mutation modules.